### ISSUE TITLE
Renaming and Instance in "Instance Manager" wipes out the settings for the original Instance you want to rename

### ISSUE THREAD

**Issue by yonkiman** [ModOrganizer2/modorganizer#2085]
## The problem:

I wanted to change just the name of an Instance (from "Fallout 4" to "Fallout London" - FOL is a mod so big and "touchy" that I'm dedicating my FO4 installation to it).  I opened **Instance Manager**, clicked on **Rename**, entered the new name "Fallout London", and _BAM_: 
- **Location** was automatically changed to C:\Users\fred\AppData\Local\ModOrganizer\Fallout London
- **Base Folder** was cleared
- I don't think I had anything in **Game** and **Game Location**, so not sure what would have happened to them

This is a problem because I didn't make a backup of the Instance, and there doesn't appear to be an automatic backup, so I have to figure it out myself.  Not a big deal, but I don't think "Renaming" an Instance should immediately wipe out all the data for that Instance, particularly with no backup.  IMO it wasn't a rename operation, it was a delete+new operation. 

## To Reproduce:

1. Open **Instance Manager**
2. Select an Instance not in use
3. Click **Rename**
4. Watch the **Location** data change and the Base Folder data disappear.

## Environment:

* Mod Organizer Version 2.5.2rc1
* Windows 10 19045.4651

**Comment by yonkiman** [comment]
Quick update: I filed that before I exited the Rename dialog.  _After_ I exited the Rename dialog, all the Instance data (Base folder, Game, Game Location) is correct except Location has been changed to match the new Name.  So only unexpected change was **Location**.

TLDR: The only issue is that Renaming an Instance automatically changes Location (and also scares you into thinking the Base folder, Game, and Game location entries have been wiped out, until you close the Rename dialog).

**Comment by yonkiman** [comment]
Update 2: Apologies for making a mess here.  I wrote this up too soon.  The only issue is what I now say in the subject: 

Renaming an Instance in "Instance Manager" _appears_ to wipe out the Base folder, Game, and Game location entries until you exit the Rename dialog.  I propose you leep the Base folder, Game, and Game location entries populate with what they will be when you exit the Rename dialog so as not to worry users.

**Comment by a2937** [comment]
Wait. I have a question. Let's say I want two profiles for Fallout 4. One for a Vanilla squeeze experience and one for Fallout London. That isn't possible is it?

**Comment by a2937** [comment]
If that isn't,  I hope that it'd be possible to make certain mods only visible to certain profiles. Like keep my dog food mod locked to London.
